x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/types.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// Separate install status type for the UI, which overlaps with common/types/InstallationStatus,
+// look into finding a way to combine them?
+export enum InstallStatus {
```

Comment:
What's the reason for not merging these two now? I don't see any conflicts?

Reply:
They need to remain separate, but they can go in the same file if we don't want public/sections having their own UI types.  InstallationStatus is for the packages in the registry and InstallStatus is different states that a package can be in in the UI.